src/loaders/lottie/tvgLottieProperty.h

```diff
@@ -560,24 +560,26 @@ struct LottiePathSet : LottieProperty
 
         auto s = frame->value.pts;
         auto e = (frame + 1)->value.pts;
+        //possible interpolation between closed and opened curves
+        auto minFrame = frame->value.ptsCnt <= (frame + 1)->value.ptsCnt ? frame : frame + 1;
```

Comment:
Logically, this sounds like unexpected behavior because it doesn't make sense to interpolate paths that don't have the same number of points. What do you think about just returning the current path without interpolation if the number of points doesn't match?

Reply:
If we do as you say, the sample json I provided will be static. I would rather keep interpolation by cutting off closing points. This change from open to closed seems suspicious, but not the need for interpolation itself. 

However, if you think it's better not to interpolate, I'll change it. 

It seems to me that overall the visual effect will be minimal and these won't be a common case

Reply:
For now, we can fix the issue with a rather simple solution I suggested. The problem is that this not only cuts off the closing but also cuts when the paths size don't match in general. Logically, This scenario couldn't be expected and we don't need to complicate things for that. Let's add a LOG now and revisit this if it really matters later by users.